src/toffy/json_utils.py

```diff
@@ -194,3 +195,40 @@ def check_for_empty_files(bin_file_dir):
 
     # return the list of fov names
     return empty_json_files
+
+
+def check_fov_resolutions(bin_file_dir, run_name):
+    """Use the run metadata to calculate the resolution of each fov
+    Args:
+        bin_file_dir (str): directory containing the run json file
+        run_name (str): name of the run and corresponding run file
+
+    Returns:
+        pd.DataFrame: details fov names and corresponding resolution value
+    """
+    # read in run metadata
+    run_file_path = os.path.join(bin_file_dir, run_name + ".json")
+    io_utils.validate_paths([run_file_path])
+    run_metadata = read_json_file(run_file_path, encoding="utf-8")
+
+    fov_names, resolutions = [], []
+    for fov in run_metadata.get("fovs", ()):
+        # get fov names
+        fov_number = fov.get("runOrder")
+        name = f"fov-{fov_number}-scan-1"
+        fov_names.append(name)
+
+        # retrieve pixel and micron specs
+        fov_pixel_length = fov.get("frameSizePixels")["width"]
```

Comment:
Is there any instance where the wouldn't have this property? If so we could add a default value as a second parameter to `.get()`.